src/SFA.DAS.ApplyService.Database/Script.PostDeployment1.sql

```diff
@@ -31,4 +31,20 @@ BEGIN
 	VALUES (NEWID(), 'Live', N'RoATPApplicationSubmittedMain', N'68008288-6497-4de2-be3c-e1409538aad9', GETDATE(), 'System')
 END
 
+-- If we TRUNCATE table, we will not need to check 'NOT EXISTS' for each entry 
+TRUNCATE TABLE [WhitelistedProviders]
```

Comment:
It might be a good idea to check for exists, as this will mean that the test environments will be reset on each deploy. We'll need to test UKPRNs that aren't in the private beta whitelist to exercise all the different types of application

Reply:
I have explained in the comment why we don’t need 'NOT EXISTS' check, but I will add it.

Reply:
It's done